### ISSUE TITLE
Slow karma tests

### ISSUE THREAD

**Issue by marcelijanowski** [karma-runner/grunt-karma#125]
Hi,
At moment I have something like 1800 tests and all stuff I taking about 10s. Is there way to speed up by splitting test to run parallel.  I have also question about autowatch because I'm running unit test in background and would be nice to only run tests file that change.
This is my current config to autowatch
```javascript
server: {
      configFile: 'test/unit/karma.conf.js',
      singleRun: false,
      autoWatch: true
    }
```

**Comment by nmccready** [comment]
Yes use https://github.com/sindresorhus/grunt-concurrent

**Comment by nmccready** [comment]
Now with the grunt-concurrent said there is a significant problem with specs being run slower now with going to the 0.12.X releases. In 0.10.1 my tests run within reason.

Please see https://github.com/karma-runner/karma/issues/1608

**Comment by nmccready** [comment]
This should be closed especially since #1608 shows the most likely problem is huge blobs.
